packages/firestore/test/unit/specs/spec_builder.ts

```diff
@@ -353,7 +358,8 @@ export class SpecBuilder {
       enableNetwork: false,
       expectedState: {
         activeTargets: {},
-        limboDocs: []
+        limboDocs: [],
```

Comment:
Done.  Note that this causes a backwards incompatibility with the JSON test specs that are generated for Android and iOS.  Once this PR is merged, I will submit PRs to those two repositories to use "activeLimboDocs" instead of "limboDocs".